File: pinyougou-ad-interface/src/main/java/com/ad/content/service/ContentService.java
package com.ad.content.service;

import com.pingyougou.pojo.TbContent;

import java.util.List;

/**
 * Created by tanwen on 2018/12/1.
 */
public interface ContentService {

    List<TbContent> findAll();

    List<TbContent> findByCategoryId(long categoryId);
}

File: pinyougou-protal-web/src/main/resources/static/js/service/ContentService.js
/**
 * Created by tanwen on 2018/12/2.
 */

app.service('contentService',function ($http) {
    
    this.findCategoryId = function (categoryId) {
        return $http.get('/content/findByCategoryId?categoryId='+categoryId);
    };
    
    
});
